### PR TITLE
TlS certificate new commands

### PR DESCRIPTION
**Pull request by IAmATeaPot418**
1. Updated how to add edit openssl ciphers
2. Updated how to add certs the new way.
3. Updated previous version references to move old way of adding your own certificate.

### CODE REVIEW

**Comment by IAmATeaPot418** [comment]
This PR resolves #377 and #434

**Comment by bmansheim** [comment]
@AlonMagrafta This is waiting for Paz' review. Can you approve it?

File: content/rs/references/procedures-previous-releases.md [hunk 1]
```diff
@@ -9,6 +9,40 @@ Here you can find procedures that were used in previous releases.
 
 ## Updating SSL/TLS certificates for Previous Releases
 
+
+## Before 5.4.4
```
Comment (AlonMagrafta) [inline content/rs/references/procedures-previous-releases.md]: change subtitle to match page's format:
## For versions below 5.4.4

File: content/rs/references/procedures-previous-releases.md [hunk 2]
```diff
@@ -9,6 +9,41 @@ Here you can find procedures that were used in previous releases.
 
 ## Updating SSL/TLS certificates for Previous Releases
 
+### Before 5.4.4
```
Comment (bmansheim) [inline content/rs/references/procedures-previous-releases.md]: @IAmATeaPot418 It looks from the words "in current versions" that this command is good for after 5.4.4 also. Can you clairfy?

Comment (IAmATeaPot418) [reply content/rs/references/procedures-previous-releases.md]: @bmansheim - yes this is good in 5.4.4 where it was initially tested.

Comment (IAmATeaPot418) [reply content/rs/references/procedures-previous-releases.md]: @Paz-Yanover - any issues with this PR?

**Comment by IAmATeaPot418** [comment]
@Paz-Yanover - just pending your approval. Everything good from your perspective?

**Comment by bmansheim** [comment]
@Paz-Yanover The procedure in https://docs.redislabs.com/staging/TLS-Certificate-NewCommands/rs/references/procedures-previous-releases/#before-5-4-4 seems to be a current way to update SSL certificates by API. Any reason why we shouldn't publish it in the regular https://docs.redislabs.com/staging/TLS-Certificate-NewCommands/rs/administering/cluster-operations/updating-certificates/?s=ssl topic?

**Comment by IAmATeaPot418** [comment]
Going to close this PR and separate it into two since we pushed half of it today but half is still outstanding. @bmansheim

**Pull request closed without merging**